dev/lsf_scripts/glwu.cron

```diff
@@ -325,14 +321,14 @@ fi
   nr_job=`qstat -wu ${USER} | grep HO 2> /dev/null | wc -l | awk '{ print $1}'`
   echo "           number of jobs     : $nr_job"
 
-  if [ "$nr_job" = '1' ]
+  if [ "$nr_job" ge '1' ]
   then
     echo ' '
     echo "           no need to run $batch ..."
     OK='no'
   fi
 
-  if [ "$nr_job" -gt '1' ]
+  if [ "$nr_job" -gt '2' ]
```

Comment:
why 2?

Reply:
In v1.1.0 the only job run is for the Great Lakes domain, so if it checks the running jobs and there are more than one running, something is wrong and it could be overwriting so it kills the jobs.
For the upgrade whenever the cronjob is submitted two jobs will be running for Great Lakes plus Lake Champlain. So the limit has been changed to 2 jobs. If one job is running, then either GL or LC cycle is not complete, so the cron should not submit a new job, and should wait (No jobs will get killed, it would simply wait). If it is greater than 2, then that is an error and it should kill the jobs.

Reply:
I see, thanks.